src/chrome/content/rules/Stubhub.mx.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+	For other Stubhub coverage, see Stubhub.com.xml
+
+
+	Non-functional subdomains:
+
+		- static	(t)
+
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Stubhub.mx">
+
+	<target host="myaccount.stubhub.mx" />
+	<target host="sell.stubhub.mx" />
+	<target host="stubhub.mx" />
+	<target host="www.stubhub.mx" />
```

Comment:
Please fix the sorting of the hosts.
LGTM otherwise.

Reply:
Fixed sorting